AppRun

```diff
@@ -1,5 +1,14 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
My general comment for the AppRun file is that it is complicated and has gone through several versions, but that it shouldn't be necessary. The appimage documentation characterizes creating a custom AppRun as a last resort, and "in most cases it is easiest (and most error-proof) to use a precompiled one from this repository." So you should be able to just use <https://github.com/AppImage/AppImageKit/releases/download/13/AppRun-x86_64>, and all of the necessary complex logic is taken care of?

Reply:
I actually didn't know there exists a default AppRun file. It should suffice so I'll use that instead.

Reply:
@doug-moen I just noticed a potentially unwanted behavior when using the precompiled AppRun. I asked about it [here](https://github.com/AppImage/AppImageKit/issues/1149).

Another thing is that it uses the `Exec` line of the embedded .desktop file to execute the AppImage. This means, if we're going to use the precompiled AppRun, which executes `Exec=curv -le %U`, then a curv app built as an AppImage and another built as a non-AppImage would have some difference in their behaviour from the command line.